example-setup/clustersetup.yaml

```diff
@@ -91,6 +100,7 @@ resources:
           flavor: { get_param: flavor_lb }
           syseleven_net: { get_resource: syseleven_net }
           public_network_id: { get_param: public_network_id }
+          mastertoken: {get_attr: [mastertoken, value]} 
```

Comment:
I think we should add dependencies for the "mastertoken:" resource to assure, that it has been created before it gets used.